### DISCUSSION TITLE
Qual melhor serviço de hospedagem

### DISCUSSION THREAD

**Discussion by silvaMatheus** [frontendbr/forum#943] [Discussão]
Pessoal estou  querendo assinar um plano de hospedagem porem nunca hospedei nenhum site e agora que vou começar hospedar alguns poderiam me indicar alguns ?

**Comment by ninetails** [comment]
Que tecnologias irá usar? Quanto estaria disposto a pagar? Isso ajudaria ao pessoal indicar o melhor serviço.

Por exemplo: se for algo em PHP, podem te indicar um hosting baseado em LAMP ou similares.

Se fizer algo em node ou dependendo da tecnologias, tem servidores de droplets e containers... Heroku, Digital Ocean, Amazon, por aí vai...

**Comment by silvaMatheus** [comment]
Bom como sou novo na área só estou fazendo sites estáticos sem nenhuma linguagem por trás, mas até junho pretendo me aprofundar muito em node, então já gostaria de pegar uma boa hospedagem,  e aproveitando o tópico indica algum lugar pra comprar domínios ?

**Comment by luizbills** [comment]
Site estático você pode ir com a https://www.netlify.com/, é gratuito e tem ótima performance. Melhor de tudo que é integrado com git, quando vc der push no seu repositório a netlify já atualiza.

**Comment by kazzkiq** [comment]
Se for apenas hospedagem sem back-end envolvido, daria pra usar serviços como [Surge](https://surge.sh/) ou [Now](https://zeit.co/now), que começam com planos free (free mesmo, sem pegadinhas).

Exemplo de como o Surge é simples:

```
$ npm install -g surge
$ cd /seus/arquivos/estaticos
$ surge
```

E só. Seu site já está hospedado.

**Comment by mahenrique94** [comment]
Se for estático dá para usar o Git Pages também.

Sobre os domínios o RegistroBR deva melhor escolha(ao meu ver).

**Comment by kvnol** [comment]
> @mahenrique94: Se for estático dá para usar o GitPages também.

Se for um freela, o cliente não vai gostar de ter o código liberado assim.

**Comment by mahenrique94** [comment]
Bom, primeiro que em nenhum momento foi mencionado ser um freela. O uso será apenas para estudos.

Segundo o problema pode ser facilmente corrigido com um repositório privado, caso não queira realizar pagamento, ai basta usar o GitLab Pages onde pode existir projetos privados sem custo.

**Comment by kvnol** [comment]
:top:

**Comment by felipefialho** [comment]
@silvaMatheus Se for só para estudos, usa o ghpages, de quebra ainda tem seu código disponibilizado, servindo para empresas verem sua evolução ;)

**Comment by MathCunhaDev** [comment]
@silvaMatheus 

Eae chará! ;D

Mano, eu gosto bastante da plataforma da Umbler, e você pode experimentar gratuitamente. 

Att.

**Comment by filipeluiz** [comment]
Olá pessoal,

Gostaria de mudar o portfolio de gh-pages para hospedagem pago, pois irei junto com outro projeto admin Vue. O que vocês acham que eu escolho um hospedagem?

**Comment by menosprezzi** [comment]
Bom dia Filipe!
Cara eu curto bastante o [Netlify](https://www.netlify.com/). Ele responde
tri bem, tem uma CDN boa, suporta HTTPS tranquilo e dá pra fazer um deploy
legal porque ele integra com o github.



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
31/01/19
11:11:00

Em qui, 31 de jan de 2019 às 10:48, Filipe Luiz <notifications@github.com>
escreveu:

> Olá pessoal,
>
> Gostaria de mudar o portfolio de gh-pages para hospedagem pago, pois irei
> junto com outro projeto admin Vue. O que vocês acham que eu escolho um
> hospedagem?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/frontendbr/forum/issues/943#issuecomment-459331950>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AF4T_TLpNLQ63hFGc1DYeEcXQS_WTXIHks5vIuYQgaJpZM4SEekz>
> .
>


-- 
*Guilherme Prezzi*

*Mobile Developer & ResearcherInstituto Federal de educação ciência e
tecnologia do Rio Grande do Sul*

**Comment by piresgraziele** [comment]
Bom dia gente! 

Bom não sei se ainda se alguém está procurando por hospedagem, mas vou deixar minha indicação do https://battlecloud.host/ . Utilizo o serviço e não tenho o que reclamar, sem contar o atendimento deles que é 24h, e o que você precisar eles estão dispostos a te ajudar. Eles possuem integração com o github também!

**Comment by NicholasKuchiniski** [comment]
paguei 50$ no meu domínio do google domains (https://nicholaskuchiniski.dev) e uso ele com o heroku para aplicações node e o github pages para sites estaticos

Bem mais barato que comprar uma hopsedagem

**Comment by cesarlopes** [comment]
Se você foi criar um site HTML ou utilizar o PHP como linguagem backend recomendo a hospedagem de site da    [www.host4.com.br](https://www.host4.com.br)

**Comment by Sanftle** [comment]
Encontrei este host há quatro meses - desde então movi meus projetos aqui https://www.melhoreshospedagemdesites.com/. Posso indicar: o preço em si é acessível e adequado para seus serviços, a velocidade e o suporte são estáveis ​​e 24 horas por dia. Além disso, durante a transferência, fiquei com medo de que houvesse problemas com a transferência, no entanto, ao entrar em contato com a equipe de suporte, eles responderam em uma hora, depois de outra hora a transferência de todos os sites foi concluída sem minha participação, tudo o que faltou foi alterar as senhas. Acho isso ótimo.

**Comment by DriCoffee** [comment]
Tem uma hospedagem boa e barata que é confiável e tem um suporte legal em português. Veja mais em [3bhost.com](http://3bhost.com)

**Comment by hugosouzadicas** [comment]
se você nunca hospedou um site, o ideal é começar com algo simples e estável, para não se complicar logo no início. Uma hospedagem compartilhada de boa qualidade já atende muito bem blogs, sites institucionais e projetos pequenos. Dê preferência a planos com SSD ou NVMe, painel de controle fácil (como cPanel) e instalador automático de sites, isso ajuda bastante quem está começando.

Evite escolher só pelo preço mais barato. Suporte decente, backups automáticos e limites razoáveis de recursos fazem muita diferença no dia a dia. Conforme você ganhar experiência ou seus sites crescerem, aí sim vale pensar em algo mais avançado, como uma VPS.

Eu mantenho um site com reviews e dicas de hospedagem justamente para quem está começando e quer escolher com mais segurança. Se quiser dar uma olhada: https://dicasdehospedagem.com.br/
